src/Main.java

```diff
@@ -1,5 +1,5 @@
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+        System.out.println("Hello world");
```

Comment:
micronit: keeping proper grammar rules is good practice for presentation. not a blocker.